src/lib/connectTraceToPlot.js

```diff
@@ -136,7 +139,7 @@ export default function connectTraceToPlot(WrappedComponent) {
               },
             });
           });
-        } else if (splitTraceGroup) {
+        } else if (splitTraceGroup && !containsAnSrc) {
```

Comment:
So the array that goes into `marker.size` also gets split in that case, so in `fullData` you see two different arrays, so it looks like Multiple Values. I added a check to prevent it, but ONLY in case when we have only ONE TRACE split into a few. That seems like the safest solution to me at the moment.

The problem with this is that, say if you have `trace 0` sized by `ints` and split into two, and `trace 1` also sized by `ints` and not split, you'll have Multiple Values show up even though they're sized with the same array. If I prevent it from comparing arrays in this case, than different arrays will look like the same value.

A more complete solution would be to pick out arrays from `_expandedInput` into the `fullData` traces just for the sake of doing the comparison in the `connectTraceToPlot.js`. It'll have to be done recursively too. I'll make a separate issue for that.

@nicolaskruchten FYI